docs/docker.md

```diff
@@ -30,41 +30,52 @@ For more information about using Docker, see the [Docker Docs](https://docs.dock
 
 ## Start the container
 
-Start a Percona Distribution for PostgreSQL container as follows:
+1. Start a Percona Distribution for PostgreSQL container as follows:
 
-```{.bash data-prompt="$"}
-$ docker run --name container-name -e POSTGRES_PASSWORD=secret -d perconalab/percona-distribution-postgresql:tag
-```
-Where:
+    ```{.bash data-prompt="$"}
+    $ docker run --name container-name -e POSTGRES_PASSWORD=secret -d perconalab/percona-distribution-postgresql:tag
+    ```    
+
+    Where:    
+
+    * `container-name` is the name you want to assign to your container
+    * `POSTGRES_PASSWORD` is the superuser password 
+    * `tag` is the tag specifying the version you want.     
 
-* `container-name` is the name you want to assign to your container
-* `POSTGRES_PASSWORD` is the superuser password 
-* `tag` is the tag specifying the version you want. 
+    Check the [full list of tags](https://hub.docker.com/r/perconalab/percona-distribution-postgresql/tags/).
+    
 
-See the list above for relevant tags, or look at the [full list of tags](https://hub.docker.com/r/perconalab/percona-distribution-postgresql/tags/).
+    !!! tip     
 
-!!! tip 
+        You can secure the password by exporting it to the environment file and using that to start the container.    
 
-    You can secure the password by exporting it to the environment file and using that to start the container.
+        1. Export the password to the environment file:    
 
-    1. Export the password to the environment file:
+            ```{.bash data-prompt="$"}
+            $ echo "POSTGRES_PASSWORD=secret" > .my-pg.env
+            ```     
 
-        ```{.bash data-prompt="$"}
-        $ echo "POSTGRES_PASSWORD=secret" > .my-pg.env
-        ``` 
+        2. Start the container:       
 
-    2. Start the container:   
+            ```{.bash data-prompt="$"}
+            $ docker run --rm -it --env-file ./.my-pg.env percona-distribution-postgresql:tag
```

Comment:
Start the container:
docker run --name container-name --env-file ./.my-pg.env -d perconalab/percona-distribution-postgresql:tag
Where:
--container-name is the name you want to assign to your container
--tag is the tag specifying the version you want.

Reply:
I think you missed this change. :)

Reply:
Indeed. Updated, thank you!